src/components/DeliverableView/DocumentsUploader.tsx

```diff
@@ -1,18 +1,33 @@
-import { useCallback, useMemo, useState } from 'react';
+import { useCallback, useEffect, useMemo, useState } from 'react';
 
 import { Box } from '@mui/material';
 import { FileChooser } from '@terraware/web-components';
 
 import { useLocalization } from 'src/providers';
+import { MAX_FILES_LIMIT } from 'src/scenes/DeliverablesRouter/DeliverableView';
```

Comment:
Move this into a constants file maybe within this directory or under `src/constants`
Ideally `scenes` depend on `components`. 
`components` are not aware of `scenes`.